packages/manager/modules/pci/src/projects/project/storages/databases/components/flavors-list/flavors-list.html

```diff
@@ -54,7 +54,7 @@
 
             <span
                 class="oui-badge oui-badge_s oui-badge_success"
-                data-ng-if="$ctrl.currentFlavor.id === flavor.id"
+                data-ng-if="$ctrl.currentFlavor.name === flavor.name"
```

Comment:
Why can't you keep `id` ?

Reply:
The id is a concatenation of both the flavor name and the plan name. Here, we want to display the badge even if the plan is different.
For exemple, if the current flavor is a db-4 with a business plan, and we choose an enterprise plan for the fork, then we want to show the "current offer" badge next to the flavor with id `enterprise-db-4`.